t/api/04-jobs.t

```diff
@@ -914,7 +914,7 @@ subtest 'handle settings when posting job' => sub {
 };
 
 subtest 'do not re-generate settings when cloning job' => sub {
-    my $job_settings = $jobs->search({test => 'autoupgrade'})->first->settings_hash;
+    my $job_settings = $jobs->find({test => 'autoupgrade'})->settings_hash;
```

Comment:
There's no unique constraint (only on a combination with other columns) so this should be limited to one row.

Reply:
Here you could specify category and script to make it unique, or just add `{rows => 1, order_by => {-desc => 'id'}}` because simply getting the first makes sense here as well.